app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -24,11 +24,18 @@ def plan(job_invocation, host, template_invocation, proxy)
 
         raise _('Could not use any template used in the job invocation') if template_invocation.blank?
 
-        settings = { :global_proxy   => 'remote_execution_global_proxy',
-                     :fallback_proxy => 'remote_execution_fallback_proxy' }
+        if proxy.blank?
+          offline_count = options[:offline_count] || 0
+          raise n_('The only usable proxy is down',
+                   'All %{number} usable proxies are down.',
+                   offline_count) % { :number => offline_count } if offline_count > 0
```

Comment:
And %{offline} would be a comma-separated list of proxy names? It would become a mess if someone had a lot of proxies, but generally I'm not opposed to providing some insight into what's going on under the hood.

Yeah, you're right 'No applicable proxy' sounds better